packages/imperative/src/logger/src/log4jsToWinston.ts

```diff
@@ -0,0 +1,273 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { transports, format, LoggerOptions, addColors } from "winston";
+import * as os from "os";
+import * as dayjs from "dayjs";
+
+export const customLevels = {
```

Comment:
I know some people may prefer `enum`s for memory efficiencies and stuff, but I think this is a clear way to access information about a specific log level 🙏 

No need to over-engineer things if we can achieve the same (or more) with just objects 😋

Reply:
Also something to consider: an enum transpiles to an object behind the scenes ^^